docs/GettingStarted.md

```diff
@@ -179,4 +179,4 @@ module.exports = {
 };
 ```
 
-However, there are some [caveats](https://babeljs.io/docs/en/next/babel-plugin-transform-typescript.html#caveats) to using TypeScript with Babel. Because TypeScript support in Babel is transpilation, Jest will not type-check your tests as they are run. If you want that, you can use [ts-jest](https://github.com/kulshekhar/ts-jest).
+However, there are some [caveats](https://babeljs.io/docs/en/next/babel-plugin-transform-typescript.html#caveats) to using TypeScript with Babel. Because TypeScript support in Babel is purely transpilation, Jest will not type-check your tests as they are run. If you want that, you can use [ts-jest](https://github.com/kulshekhar/ts-jest) instead, or setup a separate type-checking runner using [jest-runner-tsc](https://github.com/azz/jest-runner-tsc).
```

Comment:
fwiw, I'm down with the "purely" and "instead" changes if they help emphasise the difference in behavior 🙂

Reply:
Is it worth mentioning `tsc` here, or is that something so obvious? I feel it might actually get more confusing to mention `jest-runner-tsc` and then say "but it has some drawbacks, so let's use `tsc` instead", but yeah, whatever you guys think, I can make the change :)

Reply:
That's actually a good idea! Something like
```suggestion
However, there are some [caveats](https://babeljs.io/docs/en/next/babel-plugin-transform-typescript.html#caveats) to using TypeScript with Babel. Because TypeScript support in Babel is purely transpilation, Jest will not type-check your tests as they are run. If you want that, you can use [ts-jest](https://github.com/kulshekhar/ts-jest) instead, or run the TypeScript compiler separately, for example as part of your build process.
```

Reply:
@jeysal sounds good to me. Let me update the other files